src/README.md

```diff
@@ -0,0 +1,45 @@
+# Source Structure
+
+## Components
+
+`src/components`
+
+Vue components that will be used across the application reside here. These use the `vue.component` API in order to be globally available (without having to import the components each time we want to use them). Components can however contain a `components` directory themselves, for subcomponents. This can be better explained with an example:
+
+- The `fake-dialog` component is used across agriboard
```

Comment:
Maybe this should be up for discussion with the group :) ie. whether we should be nesting components